packages/tempo/src/tempo.class.ts

```
@@ -400,8 +410,8 @@ export class Tempo {
 						break;
 
 					case 'timeZone':
-						const zone = String(arg.value).toLowerCase() as keyof typeof TimeZone;
-						shape.config.timeZone = TimeZone[zone] ?? arg.value;
+						const zone = String(arg.value).toLowerCase() as enums.TIMEZONE;
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)